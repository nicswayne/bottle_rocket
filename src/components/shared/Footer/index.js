// @flow

import React from 'react';
import { Link } from 'react-router-dom';

import internets from '../../../resources/icons/tab_internets@2x.png';
import lunch from '../../../resources/icons/tab_lunch@2x.png';

const Footer = () => (
    <div className="footer">
        <Link className="button" to="/">
            <img src={lunch} alt="" />
            <div className="text">lunch</div>
        </Link>
        <Link className="button" to="/web">
            <img src={internets} alt="" />
            <div className="text">internets</div>
        </Link>
    </div>
);

export default Footer;
