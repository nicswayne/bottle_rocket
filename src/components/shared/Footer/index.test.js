import React from 'react';
import { shallow, render } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

import Footer from '.';

it('Footer component should render without crashing.', () => {
    shallow(<Footer />);
});

it('Footer component should match snapshot.', () => {
    const wrapper = render(
        <MemoryRouter>
            <Footer />
        </MemoryRouter>
    );

    expect(wrapper).toMatchSnapshot();
});
