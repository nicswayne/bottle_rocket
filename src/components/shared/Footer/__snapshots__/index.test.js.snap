// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot. 1`] = `
<div
  class="footer"
>
  <a
    class="button"
    href="/"
  >
    <img
      alt=""
      src="tab_lunch@2x.png"
    >
      
    </img>
    <div
      class="text"
    >
      lunch
    </div>
  </a>
  <a
    class="button"
    href="/web"
  >
    <img
      alt=""
      src="tab_internets@2x.png"
    >
      
    </img>
    <div
      class="text"
    >
      internets
    </div>
  </a>
</div>
`;
